engine/src/main/java/org/destinationsol/SolApplication.java

```diff
@@ -252,7 +251,7 @@ private void draw() {
             factionDisplay.drawFactionNames(solGame, uiDrawer, inputManager, solGame.getObjectManager());
         }
         if (fatalErrorMsg != null) {
-            uiDrawer.draw(uiDrawer.whiteTexture, displayDimensions.getRatio(), .5f, 0, 0, 0, .25f, 0, SolColor.UI_BG);
+            uiDrawer.draw(UiDrawer.whiteTexture, displayDimensions.getRatio(), .5f, 0, 0, 0, .25f, 0, SolColor.UI_BG);
```

Comment:
Did we figure out why this is here? Is this change useful?

Reply:
Is seems that it's a static field, so the change effectively doesn't do anything. @Adrijaned might know, though - it was in his patch.

Reply:
Probably just IDEA did something automatic